INSTALL_FROM_GITHUB.md

```diff
@@ -0,0 +1,40 @@
+## How to compile openFrameworks from github
+
+This instructions are useful just in case you want to submit a pull request or you want to have always the last openFrameworks version. In any other case, most probably, you don't need to clone the entire openFrameworks repository.
+
+If you are a developer, or if you want to submit a pull request, [read this first](https://github.com/openframeworks/openFrameworks/blob/master/README.md#developers)
+
+Let's start cloning the last `stable` branch of openframeworks.
+
+```bash
+git clone git@github.com:openframeworks/openFrameworks.git
```

Comment:
addressed in https://github.com/openframeworks/openFrameworks/commit/cfa453903c8a1f5719f50d8f2f3c2b68ace69f61